lib/fixpoint_test_helpers.rb

```diff
@@ -16,18 +16,18 @@ def restore_fixpoint(fixname, connection: default_connection)
   # ---
```

Comment:
please add documentation here (right under `+tables_to_compare+`)

Reply:
Let me know if you'd like more detail in the parameter documentation.

Reply:
❤️